tests/test_lint_utils.py

```diff
@@ -62,18 +59,7 @@ def test_run_prettier_on_malformed_file(malformed_json):
     assert malformed_json.read_text() == JSON_FORMATTED
 
 
-class TestPrettierLogging(unittest.TestCase):
-    def test_run_prettier_on_synthax_error_file(self):
-        with tempfile.TemporaryDirectory() as tmp_git_dir:
-            repo = git.Repo.init(tmp_git_dir)
-            file = Path(tmp_git_dir) / "synthax-error.json"
-            with open(file, "w", encoding="utf-8") as f:
-                f.write(JSON_WITH_SYNTAX_ERROR)
-            repo.git.add(file)
-            with self.assertLogs(level="CRITICAL") as mocked_critical:
-                nf_core.lint_utils.run_prettier_on_file(file)
-            expected_error_message = (
-                "synthax-error.json: SyntaxError: Unexpected token (1:10)\n"
-                "[error] > 1 | {'a':1, 1}\n[error]     |          ^\n"
-            )
-            assert expected_error_message in mocked_critical.output[0]
+def test_run_prettier_on_synthax_error_file(synthax_error_json, caplog):
```

Comment:
```suggestion
def test_run_prettier_on_syntax_error_file(syntax_error_json, caplog):
```

Spelling correction.